### PR TITLE
Revert #547 and #553

### PR DESCRIPTION
**Pull request by CelianR**
This reverts https://github.com/DataDog/datadog-agent-buildimages/pull/547 and https://github.com/DataDog/datadog-agent-buildimages/pull/553

### CODE REVIEW

**Review by chouetz — APPROVED** [review]

**Review by gjulianm — APPROVED** [review]
LGTM. Out of curiosity, which jobs were failing and why?

**Comment by CelianR** [comment]
@gjulianm here is a pipeline with the latest build image : https://gitlab.ddbuild.io/DataDog/datadog-agent/-/pipelines/31260218/failures. The system probe tests are failing (`aws not found`), I'm not sure why aws is not found since it is installed via awscli though

**Comment by chouquette** [comment]
> @gjulianm here is a pipeline with the latest build image : https://gitlab.ddbuild.io/DataDog/datadog-agent/-/pipelines/31260218/failures. The system probe tests are failing (`aws not found`), I'm not sure why aws is not found since it is installed via awscli though

Could this be because `echo "conda activate ddpy3" >> /root/.bashrc` is laking after the original PR? I can assist you with debugging this instead of reverting altogether if you'd like!

File: requirements.txt [hunk 1]
```diff
@@ -11,9 +11,8 @@ docker-squash
 dulwich
 invoke
 packaging
-python-gitlab
```
[the remark this answers is not in the document]
Comment (CelianR) [reply requirements.txt:14]: Indeed it is from my PR adding this package, without #547 it seems that the latest version 4.4.0 cannot be installed. I'm waiting to fix #547 before adding this

**Comment by CelianR** [comment]
FYI: This PR might be closed if https://github.com/DataDog/datadog-agent/pull/24291 fixes the issues

**Comment by CelianR** [comment]
Closing since the bug is fixed with https://github.com/DataDog/datadog-agent/pull/24291

**Pull request closed without merging**